### PR TITLE
Allow business representatives to create tasks

### PR DESCRIPTION
**Pull request by AleksiKytola**

### CODE REVIEW

File: shared/types/customTypes.ts [hunk 1]
```diff
@@ -60,6 +60,9 @@ export enum RoleType {
     Permission.VersionRead |
     Permission.RoadmapReadUsers,
   Business = Permission.TaskRead |
+    Permission.TaskCreate |
```
Comment (juhoeerola) [inline shared/types/customTypes.ts:63]: This would need a corresponding migration to update the existing business users in the database.

**Review by juhoeerola — CHANGES_REQUESTED** [review]
Noticed two bugs, otherwise looks good

File: shared/types/customTypes.ts (hunk 1 above)
Comment (AleksiKytola) [reply shared/types/customTypes.ts:63]: A good catch

File: frontend/src/components/TopBar.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const TopBar: FC<{
       />
       {children}
       <div className={classes(css.rightSide)}>
-        {role === RoleType.Admin && (
+        {(role === RoleType.Admin || role === RoleType.Business) && (
```
[the remark this answers is not in the document]
Comment (AleksiKytola) [reply frontend/src/components/TopBar.tsx:37]: I non-optionally parametrized the roles that the buttons will be shown to.

File: shared/types/customTypes.ts (hunk 1 above)
Comment (AleksiKytola) [reply shared/types/customTypes.ts:63]: Migration added. I suppose I Cant directly use RoleType.Business in the migration as future changes to that type would screw up the old migrations, so I just have 
```

const oldBusinessRole =
  Permission.TaskRead |
  Permission.TaskEdit |
  Permission.TaskDelete |
  Permission.TaskRate |
  Permission.TaskRatingEdit |
  Permission.TaskValueRate |
  Permission.CustomerRepresent |
  Permission.RoadmapReadUsers;

const newBusinessRole = oldBusinessRole | Permission.TaskCreate;

```

**Pull request merged**